src/main.rs

```diff
@@ -44,6 +44,8 @@ async fn authenticate(auth_request: Json<AuthRequest>, cookies: &CookieJar<'_>)
 	let password = &auth_request.password;
 	let seed = derive_seed(username, password, &env::var("SALT").unwrap());
 
+	let _a = username.clone();
```

Comment:
Looks like we can remove this line entirely. Maybe we should have a separate task to address any issues that could arise with formatting/refactoring.

Reply:
fixed